test/initialise.test.js

```diff
@@ -26,7 +26,10 @@ describe("initalise.js", () => {
 					data: { owner: { login: "Financial-Times" } }
 				}),
 				getInstallationRepositories: jest.fn().mockResolvedValue({
-					data: { repositories: [{ id: 12345, name: "foo-bar" }] }
+					data: { repositories: [
+						{ id: 12345, name: "foo-bar" },
+						{ id: 23456, name: "foo-bar-archived", archived: true },
```

Comment:
From https://developer.github.com/v3/activity/events/types/#installationrepositoriesevent, I'm not sure that the `archived` property exists here.

You may need to make a call to `context.github.repo.get({ name, owner })` for more information for each repository.

Or you could make a call to [`installation.apps.getInstallationRepositories`](https://developer.github.com/v3/apps/installations/#list-repositories) like we do when the app starts, as that returns the right information.

Reply:
To confirm, I tested this out by adding a removing https://github.com/financial-times-sandbox/testing-archived.

![image](https://user-images.githubusercontent.com/51677/50823965-3713a080-132d-11e9-9a77-0100fe5623fc.png)

Reply:
Curses